app/src/commonTest/java/org/mifos/mobile/RetrofitUtils.kt

```diff
@@ -10,10 +11,21 @@ import retrofit2.Response
  */
 object RetrofitUtils {
     fun get404Exception(): Exception {
-        return HttpException(Response.error<Any>(404, ResponseBody.create(MediaType.parse("application/json"), "Not Found")))
+        return HttpException(getResponseForError(404, "Not Found"))
     }
 
     fun get401Exception(): Exception {
```

Comment:
Remove this and the `get404Exception` method and directly use the `getResponseForError` method.